rigex-front-end/src/pages/Landing/Landing.js

```diff
@@ -6,6 +6,10 @@ import NavBar from '../../components/navBar/NavBar';
 import routePaths from '../../constants/routePaths';
 import validatePath from '../../utils/validatePath';
 import history from '../../utils/history';
+import RigSidebar from '../../components/RigSidebar/RigSidebar';
+import '../../styles/styles.scss';
+
+let listOfRigs = [];
```

Comment:
why is this list scoped to the file?
Is there a better place to scope your rig list? Are you using redux? Or even just scoped as an attribute on the component function?

Reply:
Now that the PR for the setup of the rigs query and rigs redux store has been submitted I was able to create and test a proper listOfRigs. This and the other comments are addressed is my next commit.